src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -2300,6 +2306,46 @@ bool ExceptionTracker::HandleNestedExceptionEscape(StackFrame sf, bool fIsFirstP
     return fResult;
 }
 
+#if defined(DEBUGGING_SUPPORTED)
+BOOL NotifyDebuggerOfStub(Thread* pThread, Frame* pCurrentFrame)
+{
+    LIMITED_METHOD_CONTRACT;
+
+    BOOL fDeliveredFirstChanceNotification = FALSE;
+
+    // <TODO>
+    // Remove this once SIS is fully enabled.
+    // </TODO>
+    extern bool g_EnableSIS;
+
+    if (g_EnableSIS)
+    {
+        _ASSERTE(GetThreadNULLOk() == pThread);
+
+        GCX_COOP();
+
+        // For debugger, we may want to notify 1st chance exceptions if they're coming out of a stub.
+        // We recognize stubs as Frames with a M2U transition type. The debugger's stackwalker also
+        // recognizes these frames and publishes ICorDebugInternalFrames in the stackwalk. It's
+        // important to use pFrame as the stack address so that the Exception callback matches up
+        // w/ the ICorDebugInternlFrame stack range.
```

Comment:
```suggestion
        // w/ the ICorDebugInternalFrame stack range.
```
Pre-existing typo (please fix other instances too)

Reply:
Fixed